nc_time_axis/tests/unit/test_NetCDFTimeConverter.py

```diff
@@ -131,7 +135,7 @@ def test_cftime_np_array_raw_date(self):
     def test_non_cftime_datetime(self):
         val = CalendarDateTime(4, '360_day')
         msg = 'The datetime attribute of the CalendarDateTime object must ' \
-              'be of type `cftime.datetime`.'
+              'be of type `cftime.datetime_base`.'
```

Comment:
This would be better using `type(cftime_datetime)` or similar to construct the `ValueError` message.